src/base/bittorrent/session.cpp

```diff
@@ -613,18 +613,18 @@ void Session::setAppendExtensionEnabled(const bool enabled)
     }
 }
 
-uint Session::stop_tracker_timeout() const
+int Session::stopTrackerTimeout() const
 {
-    return m_stop_tracker_timeout;
+    return m_stopTrackerTimeout;
 }
-void Session::setstop_tracker_timeout(const uint value)
+
+void Session::setStopTrackerTimeout(const uint value)
 {
-    if (value != stop_tracker_timeout()){
-		m_stop_tracker_timeout = value;
-	}
+    if (value != stopTrackerTimeout())
+		m_stopTrackerTimeout = value;
```

Comment:
missing indentation

Reply:
Is it fixed now? I think it was caused because by pasted the code here.